### PR TITLE
460 bug make faqs wider on mobile view

### PR DESCRIPTION
**Pull request by cherisechan**

### CODE REVIEW

**Comment by JessicaF** [comment]
@cherisechan Please remember to write a description (check Confluence page - Local Development 101) and incude any relevant screenshots!

File: frontend/__tests__/Faq.test.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+import Faq from "../components/Faq";
```
Comment (JessicaF) [inline frontend/__tests__/Faq.test.tsx:3]: Could you please check that these imports are sorted (my VSCode is telling me it isn't currently)

**Comment by JessicaF** [comment]
Good work overall! Thank you for adding the tests as well 🙏

**Review by JessicaF — CHANGES_REQUESTED** [review]

File: frontend/__tests__/Faq.test.tsx (hunk 1 above)
Comment (richard-xue-vrx) [reply frontend/__tests__/Faq.test.tsx:3]: This seems to be fine on my end, my VSC doesn't throw anything wrong here.

**Review by richard-xue-vrx — APPROVED** [review]
This looks all good to me; I'm happy with this!

File: frontend/__tests__/Faq.test.tsx (hunk 1 above)
Comment (richard-xue-vrx) [reply frontend/__tests__/Faq.test.tsx:3]: Actually that might be because I don't have a default linter on my VSC 💀

**Review by JessicaF — APPROVED** [review]

**Pull request merged**